packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -728,6 +728,7 @@ export class BackgroundScript {
         command: 'tip',
         data: {
           paymentPointer: receiver,
+          requestId: streamId,
```

Comment:
This implies the SPSP server not really doing anything relying on the Web-Monetization-Id in terms of accounting ?
It's already been deprecated right

Reply:
Should the requestId be sent to the Coil backend to send as Web-Monetization-Id to the SPSP server when sending the tip? For backend integrations that aren't yet using receipts ?

Reply:
I'm afraid I don't understand what you are asking.

Reply:
Sorry, bad sleep! I'm going to go with that :)

Depending on the WM integration, it may be that the backend tracks balances via the requestId sent to the SPSP server as "Web-Monetization-Id", so this means that in some cases, checking the balance on the backend by requestId won't show the tip. Sigh, I realize that's a bit vague, but that's because we're talking about potential integrations we can't really point to and discuss?

Reply:
It sounds like the potential issue here in most cases we have `Web-Monetization-Id` sent on the SPSP request when we make payments, and then it's emitted on the monetizationprogress event so if the server wants to correlate SPSP request/payment to the monetizationprogress event, it can.

However, Franklin doesn't send the `Web-Monetization-Id` header, so emitting a requestId on this payment may be a bit disingenuous. The site may try to cross reference that requestId with a payment only to find that no SPSP requests were made with the `Web-Monetization-Id` header.

But with the `Web-Monetization-Id` feature being deprecated (in favor of submitting receipts to verify payment) we could possibly include the requestId to avoid breaking the interface without actually sending it, with the assumption nobody will check that `Web-Monetization-Id` was really sent.

It might be better, though, just to be thorough and actually allow the `Web-Monetization-Id`/requestId to be passed to the API, then to Franklin, and then include it in Franklin's SPSP request. I don't think it would be very hard to add as an optional field and it's one less thing to worry about.

Reply:
Ok, I will hold off on merging then

Reply:
> we could possibly include the requestId to avoid breaking the interface without actually sending it

Include it in the `monetizationprogress` event without sending it via Franklin in the SPSP request? Just making sure I get it.

> It might be better, though, just to be thorough and actually allow the Web-Monetization-Id/requestId to be passed to the API, then to Franklin, and then include it in Franklin's SPSP request.

It sounds cleaner to me.

Reply:
> Include it in the monetizationprogress event without sending it via Franklin

That's basically the state of the PR now (if I'm not confusing myself)

Reply:
> Ok, I will hold off on merging then

@sharafian 
With an eye ((perhaps foolishly optimistic)) to getting a release out before the year's end?
With the fix for #1359 (and in that vein, some fireworks 31/12) and maybe #1167 now that @fruehle is back from holidays